static/templates/stream_settings/stream_settings.hbs

```diff
@@ -19,9 +19,11 @@
         <div class="general_settings stream_section">
             {{#with sub}}
             <div class="stream-header">
-                {{> stream_privacy_icon
-                  invite_only=invite_only
-                  is_web_public=is_web_public }}
+                <div class="large-icon" style="color: {{color}}">
```

Comment:
Do we need to wrap a div around these elements? The resulting HTML is kinda awkward.

Reply:
This is for 
![image](https://user-images.githubusercontent.com/63504956/127887256-6e03ab61-9f50-4575-92af-a661f3343128.png)

So in this particular case wrapping div is the colored box sorrounding that icon. So it has its own positioning logic and just different icons in it.

And most of the widget that use this icon templates have generally a span/div that handles all positioning/style logics and just has difference of icon inside it. Would it be better to keep style logic (which generally depends on places where it is used) to be separate from this?

Reply:
I see, makes sense.  I think this is probably a reasonable way to do it.